### DISCUSSION TITLE
Import Error cannot import name 'Identity' from 'paddle.nn'

### DISCUSSION THREAD

**Discussion by bvtradeoff** [PaddlePaddle/PaddleOCR#8901] [Q&A] [answered]
Team,
Am trying to processing PP-Structure, 2.2.1 image orientation + layout analysis + table recognition python script, getting Import Error: cannot import name 'Identity' from 'paddle.nn'  . 

 An issue #8890 has been logged for tracking purpose three days ago and waiting for the response. Testing Environment information provided below. Could someone please explain how to solve it.

**Training Environment.
1.Google Colab CUDA Version: 11.2
2.python 3.8
3.paddleclas 2.5.1
4.paddleocr 2.6.1.2
5.paddlepaddle 2.4.1
6.paddlepaddle-gpu 2.1.3.post112**

**Comment by Evezerest** [comment, marked as answer]
please uninstall paddlepaddle-gpu 2.1.3, and install the latest version
